src/sync/syncTranslators.js

```diff
@@ -153,4 +153,5 @@ export const SEQUENCE_KEYS = new SequenceKeyTranslator({
   requisition_serial_number: 'requisition_serial_number_for_store_',
   requisition_requester_reference: 'requisition_requester_reference_for_store_',
   stocktake_serial_number: 'stock_take_number_for_store_',
+  supplier_invoice_serial_number: 'supplier_invoice_number_for_store_'
```

Comment:
store id get's added to this values.. and mSupply desktop does some parsing 'magic'... sync api should already know which store id POSTs are coming from though...